util-core/src/main/scala/com/twitter/util/Event.scala

```diff
@@ -304,20 +304,27 @@ trait Event[+T] { self =>
   }
 
   /**
-   * Progressively build a collection of events using the passed-in
+   * Progressively build any collection of events using the passed-in
    * builder. A value containing the current version of the collection
    * is notified for each incoming event.
    */
-  def build[U >: T, That](implicit cbf: CanBuild[U, That]): Event[That] = new Event[That] {
+  def buildAny[That](implicit factory: Factory[T, That]): Event[That] = new Event[That] {
     def register(s: Witness[That]): Closable = {
-      val b = cbf()
+      val b = factory.newBuilder
       self.respond { t =>
         b += t
         s.notify(b.result())
       }
     }
   }
 
+  /**
+   * Progressively build a Seq of events using the passed-in
+   * builder. A value containing the current version of the collection
+   * is notified for each incoming event.
+   */
+  def build[U >: T, That <: Seq[U]](implicit factory: Factory[U, That]): Event[That] = buildAny(factory)
```

Comment:
add a seq-specific one, so that build that picks up any ambient factory/cbf builds at least a Seq. If we don't do this, on 2.13.0-RC1, we'll pick up a `Factory[T, Array[T]]` from the companion object of Factory, and build an `Event[Array[T]]` which is not a subtype of `Event[Seq[T]]`